doc/ideas/gameplay.md

```diff
@@ -135,6 +230,16 @@ Unit handling
  - Non-coward stance for villagers
  - Allow selecting the default stance
 
+### Viewcone-feature for additional complexity
+
+ - a viewcone shows in which direction the unit is looking at and how much it can see from that
+   - more complex Line-of-sight that this 360°-LOS
+   - e.g. a unit cannot see behind buildings -> you can hide units behind buildings from auto-attack, as long as the player is not giving manually the attack command
```

Comment:
not giving manually --> not manually giving

Reply:
done